### PR TITLE
Find contact command

### PR DESCRIPTION
**Pull request by Ma-Xizi**

### CODE REVIEW

File: src/main/java/tutorease/address/logic/commands/FindContactCommand.java [hunk 1]
```diff
@@ -8,24 +8,26 @@
 import tutorease.address.model.person.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
- * Keyword matching is case insensitive.
+ * Finds all persons whose name is contained in the keyword.
  */
-public class FindCommand extends Command {
+public class FindContactCommand extends Command {
+    public static final String COMMAND_WORD = "contact";
```
Comment (ethantyh692) [inline src/main/java/tutorease/address/logic/commands/FindContactCommand.java:14]: shouldn't there be a class for ContactCommand where there is a ContactCommand.COMMAND_WORD that is "contact"? can reference the lesson side

File: src/main/java/tutorease/address/logic/commands/FindContactCommand.java [hunk 2]
```diff
@@ -8,24 +8,26 @@
...
 
-    public static final String COMMAND_WORD = "find";
+    public static final String SUB_COMMAND_WORD = "find";
```
Comment (ethantyh692) [inline src/main/java/tutorease/address/logic/commands/FindContactCommand.java:16]: won't need a SUB_COMMAND_WORD if follow the suggestion above. will just be ContactCommand.COMMAND_WORD and COMMAND_WORD

File: src/main/java/tutorease/address/logic/commands/FindContactCommand.java [hunk 3]
```diff
@@ -8,24 +8,26 @@
...
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + SUB_COMMAND_WORD
```
Comment (ethantyh692) [inline src/main/java/tutorease/address/logic/commands/FindContactCommand.java:18]: MESSAGE_USAGE should change accordingly

**Review by ethantyh692 — COMMENTED** [review]
Other than the changes suggested, LGTM

File: src/main/java/tutorease/address/logic/commands/FindContactCommand.java (hunk 1 above)
Comment (Ma-Xizi) [reply src/main/java/tutorease/address/logic/commands/FindContactCommand.java]: Fixed

**Review by NorbertLoh — APPROVED** [review]
LGTM!

File: src/main/java/tutorease/address/logic/commands/AddContactCommand.java [hunk 4]
```diff
@@ -18,11 +18,10 @@
...
  */
 
-public class AddContactCommand extends Command {
-    public static final String COMMAND_WORD = "contact";
-    public static final String SUB_COMMAND_WORD = "add";
+public class AddContactCommand extends ContactCommand {
+    public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + " " + SUB_COMMAND_WORD
+    public static final String MESSAGE_USAGE = ContactCommand.COMMAND_WORD + " " + COMMAND_WORD
```
Comment (Chronoxy) [inline src/main/java/tutorease/address/logic/commands/AddContactCommand.java:24]: Good call on realizing the inconsistency and fixing it 👍

**Review by Chronoxy — APPROVED** [review]
LGTM 👍

**Pull request merged**